### DISCUSSION TITLE
Moes MS-104BZ Pairing

### DISCUSSION THREAD

**Discussion by alberto072710** [Koenkk/zigbee2mqtt#7634] [Devices]
Good afternoon,

Someone has made the Moes MS104-BZ work, I try to put it in pairing mode by turning the light off and on 10 times but nothing appears in the debug.  The beep sound is heard continuously for a few seconds and turns off Any ideas?

Thanks.

**Comment by c0mplex1** [comment]
Hi,

After trying several times I managed to pair the device.
Follow the steps in the user manual very carefully.
I recommend to reset the device first and then try pairing again.
To do this, follow the steps described on pages 17/18 of the manual, point 2.3 (clear up the RF code) and then point 1.3 (pair the RF code) .
![afbeelding](https://user-images.githubusercontent.com/81088928/120100887-754ecc00-c143-11eb-9f17-df490a3995e5.png)

**Reply by alberto072710** [reply]
Thank you very much for your answer. I have tried to do what you told me but no luck. Are you sure it's page 17/18 of the manual? Isn't that to match it via RF? I thought that to put it in pairing mode it is page 10.

**Reply by c0mplex1** [reply]
Maybe that's why I had some difficulties to pairing the device.
Pairing described on page 10 is much easier, and it looks like it's for ZigBee devices, not for RF.
Maybe I have by accident followed the steps as described on page 10, I don't know.
Suddenly the device was paired.

**Reply by alberto072710** [reply]
Thank you very much for your help.
Do you remember if before pairing the device kept ringing continuously? Is that when I follow the pairing steps, the equipment makes 20 beeps and shuts up.... According to the manual it should sound continuously.
It drives me crazy.
Thank you

**Reply by c0mplex1** [reply]
Sorry, but I can't say for sure if the device sounds a long beep.
What I remember is, it makes a lot of noise (sound) during pairing mode, and I was glad it suddenly pairs.
Some settings had I do twice, one for gang-l1 and one for gang-l2. That's why I said read page 17/18.
Maybe that makes it so difficult.

**Reply by eruizman** [reply]
Hi Alberto, I've same problem as you.... Finally, were you able to pair it? 
Esteban

**Reply by alberto072710** [reply]
Hi,
I have not been able to match it. I finally gave up and put it in a drawer...

**Comment by ThermIQ** [comment]
I've managed to get this device to pair by using the built-in reset button. After a lot of struggling, this is what I did:
1. Short Push and release reset button -> 1 Beep
2. Push and hold reset button until Beep, Beep, Release reset button
3. Push and hold reset button, wait for beep, beep, and continue hold until device starts to short beep continously. (Beep, beep, beep, ...)
Device is now in pairing mode

Notes:
- I had to turn of my IKEA Trådfri Gateway
- Manual is confusing as it describes 3 different ways of reset and pair, by Switch button (external), by Rocker button(external) and by Reset button (internal)
- This worked on the two devices I have, but I have not dared to reset and pair them again :-)

**Reply by pmknowles** [reply]
I've got mine to pair by press and hold reset for about 15 seconds until it beeps twice. Then press and hold for about 10 seconds. It's not fool proof though

**Reply by ThermIQ** [reply]
Thanks, sounds more or less the same as what I did except step 1. It might be that it is not needed but who knows, these are a bit tricky for sure :-)

**Comment by polgarc** [comment]
This is what finally worked for me with the reset button:

1. Short push and release -> 1 beep
2. Push and hold -> 2 beeps
3. Don't release, keep pushing -> continuous beeping
4. Release, device is in pairing mode

**Reply by ThermIQ** [reply]
This is consistent with my latest installed device

**Reply by michel-zedler** [reply]
The manual on page10 has incorrect instructions:

> **Zigbee Link/Reset**
> (...)
> **c. For reset button:** long press the button on the module to hear the beep sound as Di-Di(2 times) and long press the button again to hear the beep sounds continuously.

This does not link and seems to perform the RF reset described on page 18 instead.

The method described by @polgarc works! Many thanks!

**Reply by chrisys** [reply]
This method worked for me!

**Reply by Puntoboy** [reply]
this method worked for me as well. It's been sat in a draw for over a year as I couldn't get it paired the first time, so thanks.

**Comment by Autogarsas** [comment]
So i had trouble with pairing mode too, and found what was a problem (probably for all of us)
You need to press switch 10 times fast (reset switch)
And if rocker switch you need to do that fast 10 times each direction.
The relay don't need to click, need to do that fast, that was mistake.

**Reply by EddyGitH** [reply]
I was struggling too with pairing. I have a version with reset button on the device, that didn't work for me. Beeps(2) yes, but no continuous beeping.  Used a switch mounted to the device and flipped it a lot and then 2 beeps and after continuing beeps continously.

**Reply by crisapat** [reply]
Hi do you able to pair it using RF? What frequency must be used?

**Comment by pmknowles** [comment]
I just had to re-pair a device with a reset button on (the Moes version).
Pressed the reset button and released. Pressed again and held until the rapid continuous beeping. Paired first time.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by pmknowles** [comment]
Well you are asking a question about the MS105BZ on a thread about the MS104.
Try powering the device off/on 5 times

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by dpolyakov** [comment]
Hello.

Can`t pair Moes MS-104BZ with SONOFF Zigbee_3.0 USB Dongle Plus V2

Tried method 
```
Short push and release -> 1 beep
Push and hold -> 2 beeps
Don't release, keep pushing -> continuous beeping
```
here log from z2m
```log
Info 2023-08-26 17:27:23MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"left_network","meta":{"friendly_name":"0x847127fffe8daf01"},"type":"device_removed"}'
Warning 2023-08-26 17:27:23Device '0x847127fffe8daf01' left the network
Info 2023-08-26 17:27:24MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x847127fffe8daf01","ieee_address":"0x847127fffe8daf01"},"type":"device_leave"}'
Info 2023-08-26 17:27:24MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"left_network","meta":{"friendly_name":"0x847127fffe8daf01"},"type":"device_removed"}'
Warning 2023-08-26 17:27:24Device '0x847127fffe8daf01' left the network
Info 2023-08-26 17:27:25MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x847127fffe8daf01","ieee_address":"0x847127fffe8daf01"},"type":"device_leave"}'
Info 2023-08-26 17:27:25MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"left_network","meta":{"friendly_name":"0x847127fffe8daf01"},"type":"device_removed"}'
Warning 2023-08-26 17:27:25Device '0x847127fffe8daf01' left the network
Info 2023-08-26 17:27:25MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x847127fffe8daf01","ieee_address":"0x847127fffe8daf01"},"type":"device_leave"}'
Info 2023-08-26 17:27:25MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"left_network","meta":{"friendly_name":"0x847127fffe8daf01"},"type":"device_removed"}'
Warning 2023-08-26 17:27:25Device '0x847127fffe8daf01' left the network
```

**Comment by pmknowles** [comment]
It's 433MHz.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by crisapat** [reply]
I tried to pair it with broadlink rm4c pro that supports 433 MHz but not working. Do you know the specific 433Mhz it supported?

**Reply by pmknowles** [reply]
The devices with RF433 can only receive on/off signals from RF433 push button type devices. You press the required button on the Moes device for 5 seconds. Then press the remote button. If you look at the relevant product page on the Moes website there's a link to 'Help file' below the specifications link. That will allow you to download the manual as a pdf.
